### PR TITLE
.coafile: Add

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
@sils1297 not sure if this is what you wanted with the coafile

**Comment by sils** [comment]
ack https://github.com/coala-analyzer/coala-decorators/pull/13/commits/aef232d4c5b865c7aee72052d3817f4296ac00fc

**Comment by sils** [comment]
can you make another commit adding Radon and CPD?

File: coala_decorators/.coafile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[Default]
+files = *.py, ./coala, tests/**/*.py
```
Comment (sils) [inline coala_decorators/.coafile]: oh this glob is totally wrong

**Comment by sils** [comment]
unack https://github.com/coala-analyzer/coala-decorators/pull/13/commits/aef232d4c5b865c7aee72052d3817f4296ac00fc

**Comment by Adrianzatreanu** [comment]
i believe this is the only glob we can have right now?

**Comment by sils** [comment]
ack https://github.com/coala-analyzer/coala-decorators/pull/13/commits/9b22d28f2d6d9344b97209845cb44221c961fe8b https://github.com/coala-analyzer/coala-decorators/pull/13/commits/959fed1f155f587d592fa9536fc7c86b7f9f4bad https://github.com/coala-analyzer/coala-decorators/pull/13/commits/3fb22fef903cfc8fb10cab4a86d407c95b427e8d

**Comment by sils** [comment]
@rultor merge

**Pull request merged**